app/src/main/java/org/cuberite/android/MainActivity.java

```diff
@@ -152,6 +150,14 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
             }
         }
     }
+    
+    @Override
+    public void onPause() {
+        super.onPause();
+        if (permissionPopup != null) {
+            permissionPopup.dismiss();

```

Comment:
Also set `permissionPopup = null` here, otherwise we might dismiss a popup twice...